cmd/version.go

```diff
@@ -15,11 +17,33 @@ var versionCmd = &cobra.Command{
 	Long:  `Shows the Cobbler server version.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		generateCobblerClient()
-
-		// TODO: call cobblerclient
+		version, err := Client.ExtendedVersion()
+		if err != nil {
+			fmt.Println(err)
+		}
+		clientVersion, cliVersion, _ := getClientVersion()
+		fmt.Printf("Cobbler %s\n", version.Version)
+		fmt.Printf("  source: %s, %s\n", version.Gitstamp, version.Gitdate)
+		fmt.Printf("  build time: %s\n", version.Builddate)
+		fmt.Printf("  cli: %s\n", cliVersion)
+		fmt.Printf("  client: %s\n", clientVersion)
 	},
 }
 
+func getClientVersion() (string, string, error) {
+	bi, _ := debug.ReadBuildInfo()
```

Comment:
The only way I can find is (`ReadBuildInfo`)[https://pkg.go.dev/runtime/debug#ReadBuildInfo]. If you can show me another one I'd be happy to oblige.

Reply:
I don't think it's that much better, but one way would be to read the Client version from the lock file. I just can't believe that golang has no clean way to get versions of dependencies, because these feel very hacky to me

Reply:
the `debug` package is a part of the `runtime` package, it's not related to "debugger" or "debugging", it's the official, non-hacky way, to get information about dependencies.